solutions/devsprint-leonardo-santos-3/FinanceAppTests/TestUtils/Doubles/NetworkClientSpy.swift

```diff
@@ -1 +1,17 @@
+import Foundation
 
+@testable import FinanceApp
+
+class NetworkClientSpy: NetworkClientProtocol {
+    // MARK: Properties
+    var url: URL?
+    var completionData: Data?
+    var performRequestCount: Int = 0
```

Comment:
O que você acha de mantermos as propriedades passadas como `private(set)`? Dessa forma evitamos que os devs coloquem o valor na mão, podendo assim forçar o resultado dos testes

```suggestion
    private(set) var url: URL?
    private(set) var performRequestCount: Int = 0
    var completionData: Data?
```